compiler/ast/parser.nim

```diff
@@ -2004,14 +2036,14 @@ proc parseTypeDef(p: var Parser): ParsedNode =
         (p.emptyNode, p.emptyNode) # xxx: error?
 
   result.add:
-    if pragmas.kind == nkEmpty:
+    if pragmas.kind == pnkEmpty:
       identifier
     else:
-      p.newNode(nkPragmaExpr, @[identifier, pragmas])
+      p.newNode(pnkPragmaExpr, p.tok, @[identifier, pragmas])
   result.add genericParams
   result.add case p.tok.tokType
              of tkEquals:
-               result.info = p.lineInfo
+               result.token = p.tok # TODO: before merge confirm this makes sense
```

Comment:
It does, in the sense that the `=` of the type declaration will be considered the root of it, similar to a binary operator.

Reply:
Fair enough, I noted some of the conventions in the reply to the prior comment, I think the gap is a sensible specification. 😆

Reply:
Removed the comment as confirmation.